0007/README.md

```diff
@@ -0,0 +1,225 @@
+    *	SC Members share updates on their activities in the Community Meeting.
+
+### Clarify GitHub roles and how they maps to project roles
+
+Currently it’s unclear how the GitHub org and repo membership and roles
+correspond to the roles defined in the governance document. The proposal here
+attempts to clarify.
+
+*   GitHub Organization:
+    *   Member:
+        *   Used for Maintainers to allow them to be added to a team.
+        *   Used as needed for Participants and Contributors who would like to
+            set up [custom notification routing] and issue assignment across
+            multiple repos.
+        *   Does not grant any special repository permissions, i.e. Base
+            permissions = “No permission”
+    *   Owner:
+        *   Used for SC Members, OpenSSF administrators, and (as needed)
+            Maintainers
+    *   Teams: (all with public visibility)
+        *   [SLSA Steering Committee][SLSA Steering Committee Team]
+        *   Core Maintainers
+        *   \<subproject\> Maintainers
+        *   Other teams as needed
+    *   GitHub Repositories
+        *   Read: may be granted to Participants and Contributors as needed,
+            usually to allow the person to set up custom notification routing
+            and/or be assigned issues
+        *   Triage/Write: may be granted to Contributors who have been
+            delegated this responsibility by a Maintainer
+            *   Write permission grants ability to approve PRs
+        *   Admin: granted to every Maintainer via the \<subproject\>
+            Maintainers team
+    *   These permissions should be reviewed and cleaned up regularly (e.g.
+        annually)
+
+Once this proposal is accepted, the following changes will be made to the
+GitHub organisation:
+
+*   Set up Maintainers teams and add Maintainers as members if needed
+*   Change the org member base permissions to “No permissions” (from “Admin”)
+    and check that everyone still has necessary access
+*   Purge unnecessary permissions in the org and in each repo
+	*	Purging of unnecessary permissions should happen on a regular cadence. At minimum
+		after each SC Member nomination cycle.
+
+### Other clarifications
+
+*   Add SC Member duty: "selecting Maintainers and Steering Committee Members"
+*   Add SC Member duty: "representing SLSA in the OpenSSF SCI WG"
+*   Remove requirement for SC members to meet "no less then [sic] once per
+    quarter".
+*   Add requirement to publish minutes for every SC meeting.
+*   State explicitly that SLSA lives within and takes broad direction from the
+    OpenSSF SCI WG.
+
+## Alternatives considered
+
+*   Status quo
+    *   see [goals]
+*   Dissolve the SLSA Steering Committee in favor of steering from the SCI WG
+    *   Con: Strategy may be driven by those without deep familiarity with
+        the project.
+    *   Con: SLSA community may not feel an appropriate sense of ownership and
+        autonomy.
```

Comment:
For the sake of completeness this should include the pro:
```suggestion
*   Dissolve the SLSA Steering Committee in favor of steering from the SCI WG
    *   Pro: simplified governance structure
    *   Con: Strategy may be driven by those without deep familiarity with
        the project.
    *   Con: SLSA community may not feel an appropriate sense of ownership and
        autonomy.
```

Reply:
Agreed, added in [4432b4a](https://github.com/slsa-framework/slsa-proposals/pull/15/commits/4432b4a8dd41a8ee2ab73f1a074b409ab7e4e07e). Thanks!